src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs

```diff
@@ -17,17 +18,48 @@ public ResumesService(IRepository<Resume> resumesRepository)
 
         public Resume GetResume(Guid id)
         {
-            return this.resumesRepository.GetById(id);
+            try
+            {
+                var resume = this.resumesRepository.GetById(id);
+                if (resume == null)
+                {
+                    throw new ResumeException(404);
+                }
+
+                return resume;
+            }
+            catch (ResumeException ex)
+            {
+                throw ex;
+            }
```

Comment:
I believe this catch is not necessary

Reply:
It is necessary to catch the exception of ResumeException type

Reply:
but it will be caught at the function level